src/components/CitiesWeatherWidget/components/Title.tsx

```diff
@@ -0,0 +1,11 @@
+import { Name } from './components.styled';
+
+interface Props {
+  name: string;
+}
+
+const Title = ({ name }: Props) => {
+  return <Name>{name}</Name>;
+};
```

Comment:
**suggestion** i think it is unnecessary to create whole component just to add styling. I would suggest to just use `Name` styled component in CitiesWeatherWidget like this `CitiesWeatherWidget.Title = Name;`

Reply:
also the same can be done to `ShortcutWrapper`

Reply:
Good point :)